HARK/ConsumptionSaving/ConsMarkovModel.py

```diff
@@ -338,39 +338,44 @@ def vpp_next(shocks, a_nrm):
         )
         return EndOfPrdvPP
 
-    def make_EndOfPrdvFuncCond(self):
+        
+    def make_EndOfPrdvFuncCond(self, EndOfPrdvP):
```

Comment:
Shouldn't require EndOfPrdvP as argument. Can reference it as self as it is computed earlier as stored as an attribute.